### ISSUE TITLE
[FEATURE] Streamer Mode

### ISSUE THREAD

**Issue by CorruptedSeal** [RusherDevelopment/rusherhack-issues#378]
**What feature would you like to be added.**
Randomly scramble (add/multiple/divide?) the players coordinates to throw off players tracking their movements while streaming.

**Is your feature request related to a problem? Please describe.**
N/A

**Additional context**
N/A

**Comment by FluorideHistorian** [comment]
just turn off your coords hud?

**Comment by John200410** [comment]
I plan to add a streamer mode setting that does this and also replaces the bedrock texture.

**Comment by tycrek** [comment]
Similar to #90

**Comment by ItzDawnfall** [comment]
If a streamer mode is being added, I really also having it turn off sign text.

**Comment by John200410** [comment]
> If a streamer mode is being added, I really also having it turn off sign text.

You can already disable this under NoRender but I will probably also have an option in StreamerMode or just move it to this module.

**Comment by ItzDawnfall** [comment]
Yeah, it would be nice to have everything related to that be under one module. Maybe you could also have an option under StreamerMode to hide wither names as well to prevent having offensive wither names on stream

**Comment by John200410** [comment]
Well at that point you are probably looking for something like this https://github.com/rebane2001/2spicy2stream

**Comment by CaptainTheBeam** [comment]
I like this can you also add a unmute captain module?

**Comment by John200410** [comment]
https://github.com/kybe236/rusherhack-streamer-mode
